snuba/web/split.py

```diff
@@ -15,7 +15,7 @@
 
 def split_query(query_func):
     def wrapper(dataset, request: Request, *args, **kwargs):
-        use_split = state.get_configs([("use_split", 0)])
+        use_split, = state.get_configs([("use_split", 0)])
```

Comment:
I just realised setting this to 0 didn't actually do anything

Reply:
Yikes, good catch.